sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py

```diff
@@ -6,19 +6,25 @@
 
 import base64
 import json
-import time
 from typing import (  # pylint: disable=unused-import
     cast,
     Tuple,
 )
 from datetime import datetime
+import calendar
```

Comment:
This file will need to be updated across all the communications libraries (which all have the _shared module)

Reply:
I have had imported